sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -228,7 +228,7 @@ export interface PeekMessagesOptions extends OperationOptions {
 // @public
 export interface QueueDescription {
     authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
+    autoDeleteOnIdleInSeconds?: number;
```

Comment:
Wasnt the guidance to use milli seconds in general in our JS SDKs? What data type are other languages using?

Reply:
I did think of it, but usual values for this would be in "days", did not make much sense to convert days into milliseconds.

Reply:
.NET has a special TimeSpan type for durations.
- https://docs.microsoft.com/en-us/dotnet/api/system.timespan?view=netcore-3.1

Java is relying on a similar Duration type
- https://github.com/Azure/azure-sdk-for-java/blob/3073499e73ca0d9d272942454f75819ecaf55330/sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/QueueDescription.java#L51

Reply:
It seems fine to me to use seconds for 2 reasons:
1. I _think_ the ISO 8601 duration format only goes down to the second precision (though it does allow fractional seconds.)
2. arm-servicebus has these properties as well (without the `InSeconds` suffix) as strings. If the fields are named differently, then I think there will be less confusion about why one package returns a string and another returns a number.

Reply:
Assuming that these properties need to work with larger timeframes like years to smaller values of seconds, are we sure that dealing with numbers is indeed more user friendly than using the ISO-8601 format?

One requires to do math, the other requires prior knowledge of ISO-8601 format.

Given that arm package will not be changing, is it worthwhile having separate treatment of these properties?

Reply:
> One requires to do math, the other requires prior knowledge of ISO-8601 format.

I had the same dilemma while porting the tests in this PR.

> are we sure that dealing with numbers is indeed more user friendly than using the ISO-8601 format?

I would prefer ISO-8601 duration format over numbers. I don't know if it is conventional or convenient for the users. 
If we prefer ISO-8601 duration format, we can add the [ISO_8601#Durations](https://en.wikipedia.org/wiki/ISO_8601#Durations) link in the docs for all the attributes that take ISO-8601 string inputs.   
This would also be consistent with the arm package.

Reply:
Ah, I assumed that this had already been decided in an API review. If not then I'm also fine with keeping it as a string. When we get `Temporal` we can look at supporting that in the future.

Reply:
The call to change this was quickly made in the .Net review, but I believe that was because they have a very convenient class `Duration` that is more user friendly than the ISO format.

Reply:
cc @bterlson, @xirzec

Reply:
After reflecting on this a bit more, it might be best to leave these as ISO strings until we get a better duration primitive to work with in JS.